LPS-181508

simonjhy opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

simonjhy commented:
@petershin I changed some code and added the .m2-tmp as a repository for test, It could pass all project template test now. Here is test result(https://github.com/simonjhy/liferay-portal/pull/158), I checked the test log, the test aborted is not our issue, our template has been already finsihed.
About the exclude rules, your changes can work in loca and all tests can pass and it always failded in CI server, I not sure what' problem, but I guessed that the CI server start test not matched our condiftion, so it did not include the .gitignore in workspace template, so I commented other code(https://github.com/petershin/liferay-portal/pull/2092/commits/aae37d8eb6ff4b9424c2d6c226b3f0513089ddfc)
